examples/basic/main.tf

```diff
@@ -22,8 +29,46 @@ resource "talos_machine_configuration_worker" "machineconfig_worker" {
   machine_secrets  = talos_machine_secrets.machine_secrets.machine_secrets
 }
 
+resource "talos_client_configuration" "talosconfig" {
+  cluster_name    = var.cluster_name
+  machine_secrets = talos_machine_secrets.machine_secrets.machine_secrets
+  endpoints       = ["10.5.0.2"]
+}
+
+// use data source if you want a get a new talos client config each time TF is run
```

Comment:
not sure if we should just use resources only or support both `data` and `resource`. Data source means a new client config is generated on each run

Reply:
plan to deprecate the data sources at a later point